src/participant_network/cl/prysm/prysm_launcher.star

```diff
@@ -0,0 +1,291 @@
+		cmd_args.append("--bootstrap-node="+bootnode_context.enr)
+
+	if mev_boost_context != None:
+		cmd_args.append(("--http-mev-relay{0}".format(mev_boost_endpoint(mev_boost_context))))
+
+	if len(extra_params) > 0:
+		# we do the for loop as otherwise its a proto repeated array
+		cmd_args.extend([param for param in extra_params])
+
+	return struct(
+		image = beacon_image,
+		ports = BEACON_NODE_USED_PORTS,
+		cmd_args = cmd_args,
+		files = {
+			genesis_data.files_artifact_uuid: GENESIS_DATA_MOUNT_DIRPATH_ON_SERVICE_CONTAINER,
+		},
+		privaite_ip_address_placeholder = PRIVATE_IP_ADDRESS_PLACEHOLDER
+	)
+
+
+def get_validator_config(
+		genesis_data,
+		validator_image,
+		service_id,
+		log_level,
+		beacon_rpc_endpoint,
+		beacon_http_endpoint,
+		node_keystore_files,
+		mev_boost_context,
+		extra_params,
+		prysm_password_relative_filepath,
+		prysm_password_artifact_uuid
+	):
+
+	consensus_data_dirpath = path_join(CONSENSUS_DATA_DIRPATH_ON_SERVICE_CONTAINER, service_id)
+	prysm_keystore_dirpath = path_join(VALIDATOR_KEYS_MOUNT_DIRPATH_ON_SERVICE_CONTAINER, node_keystore_files.prysm_relative_dirpath)
+	prysm_password_filepath = path_join(PRYSM_PASSWORD_MOUNT_DIRPATH_ON_SERVICE_CONTAINER, prysm_password_relative_filepath)
+
+	cmd_args = [
+		"--accept-terms-of-use=true",#it's mandatory in order to run the node
+		"--prater",                  #it's a tesnet setup, it's mandatory to set a network (https://docs.prylabs.network/docs/install/install-with-script#before-you-begin-pick-your-network-1)
+		"--beacon-rpc-gateway-provider=" + beacon_http_endpoint,
+		"--beacon-rpc-provider=" + beacon_rpc_endpoint,
+		"--wallet-dir=" + prysm_keystore_dirpath,
+		"--wallet-password-file=" + prysm_password_filepath,
+		"--datadir=" + consensus_data_dirpath,
+		"--monitoring-port={0}".format(VALIDATOR_MONITORING_PORT_NUM),
+		"--verbosity=" + log_level,
+		# TODO(old) SOMETHING ABOUT JWT
+		# vvvvvvvvvvvvvvvvvvv METRICS CONFIG vvvvvvvvvvvvvvvvvvvvv
+		"--disable-monitoring=false",
+		"--monitoring-host=0.0.0.0",
+		"--monitoring-port={0}".format(VALIDATOR_MONITORING_PORT_NUM)
+		# ^^^^^^^^^^^^^^^^^^^ METRICS CONFIG ^^^^^^^^^^^^^^^^^^^^^
+	]
+
+	if mev_boost_context != None:
+		# TODO(old) required to work?
+		# cmdArgs = append(cmdArgs, "--suggested-fee-recipient=0x...")
+		cmd_args.append("--enable-builder")
+
+
+	if len(extra_params) > 0:
+		# we do the for loop as otherwise its a proto repeated array
+		cmd_args.extend([param for param in extra_params])
+
+
+	return struct(
+		image = validator_image,
+		ports = VALIDATOR_NODE_USED_PORTS,
+		cmd_args = cmd_args,
+		files = {
+			genesis_data.files_artifact_uuid: GENESIS_DATA_MOUNT_DIRPATH_ON_SERVICE_CONTAINER,
+			node_keystore_files.files_artifact_uuid:             VALIDATOR_KEYS_MOUNT_DIRPATH_ON_SERVICE_CONTAINER,
+			prysm_password_artifact_uuid:          PRYSM_PASSWORD_MOUNT_DIRPATH_ON_SERVICE_CONTAINER,			
+		},
+		privaite_ip_address_placeholder = PRIVATE_IP_ADDRESS_PLACEHOLDER
```

Comment:
`s/privaite/private`

Reply:
btw all of these continued to work as my PRIVATE_IP_ADDRESS_PLACEHOLDER was the default. We should fail loudly for struct keys that shouldn't be there. I thi